tests/turbo.node.test.ts

```diff
@@ -227,18 +239,57 @@ describe('Node environment', () => {
         expect(winc).to.be.a('string');
       });
     });
+
+    describe('submitFundTransaction()', () => {
```

Comment:
project nit: most of these tests are identical. could we consolidate them into functions and setupp each env then run each test suite function?

Reply:
Yea if there were a way we could factor out the common pieces and use them and inject the platform specific pieces that would be great.

Reply:
agree - but the types are different per env - i.ee web tests post ReadbleStreams and node tests post Readables - I'll make a ticket to look at there's a multi-env testing pattern we can use for web vs. node